api/app/src/routes/medical/document.ts

```diff
@@ -66,6 +66,24 @@ router.get(
 /** ---------------------------------------------------------------------------
  * GET /document/query
  *
+ * Returns the document query status for the specified patient.
+ *
+ * @param req.query.patientId Patient ID for which to retrieve document query status.
+ * @return The status of document querying across HIEs.
+ */
+router.get(
+  "/query",
+  asyncHandler(async (req: Request, res: Response) => {
```

Comment:
nit: would `/queryStatus` be better?

Reply:
I'd keep `/query` to be more RESTul. One POSTs and GETs on the same endpoint, just changing the method/operation one changes what's being done.

Also, the response for both operations on the same endpoint are slightly the same, which makes it nice go just swap the method.